cypress/plugins/bdd.ts

```diff
@@ -1,12 +1,35 @@
 const browserify = require('@cypress/browserify-preprocessor');
 const cucumber = require('cypress-cucumber-preprocessor').default;
 const path = require('path');
+const axios = require('axios');
+const default_config = require('../../cypress-bdd.json')
+
 
 module.exports = (on, config) => {
+  config.env.baseUrl = process.env.CYPRESS_BASE_URL || default_config.baseUrl
```

Comment:
By default cypress has `config.baseUrl` option. You can override this by env var or command line option: `cypress --config baseUrl=http://mycluster.crc/kiali`. IMO providing the URL via command line is the preferred way to run cypress against different environments. Importing the bdd json directly does not allow you to do this. If you remove this line and the import on line 5 then on line 29 use `config.baseUrl` you'll get the same functionality plus keep the ability to override via cmd line.

Example running cypress via cmd line:
```
yarn cypress:bdd --config baseUrl="https://localhost:20001" --env username=testuser,password=xxxx
```

Reply:
funny, this works with YARN OK as you mentioned, but not with NPM
`yarn cypress:bdd --config baseUrl="https://localhost:20001" --env username=testuser,password=xxxx`